### PR TITLE
DYN-5996 Project migrations

### PR DESCRIPTION
**Pull request by sm6srw**
### Purpose
This pull request migrates:
* GeometryUIWpf
* DataBridgeTests
* WorkflowTests
* CommandLineTests
* UnitsUI
* TestUINodes
* CoreNodeTests

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [x] This PR contains no files larger than 50 MB 

### Release Notes

(FILL ME IN) Brief description of the fix / enhancement. **Mandatory section** 

### Reviewers

(FILL ME IN) Reviewer 1  (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/DynamoCoreWpf/DynamoCoreWpf.csproj [hunk 1]
```diff
@@ -158,12 +158,16 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\extern\Microsoft.Xaml.Behaviors\$(TargetFramework)\Dynamo.Microsoft.Xaml.Behaviors.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Practices.Prism">
+      <HintPath>..\..\extern\prism\Microsoft.Practices.Prism.dll</HintPath>
```
Comment (aparajit-pratap) [inline src/DynamoCoreWpf/DynamoCoreWpf.csproj:162]: I thought @pinzart90 removed this.

Comment (mjkkirschner) [reply src/DynamoCoreWpf/DynamoCoreWpf.csproj:162]: good catch, I think we should reference the new versions Tibi brought in.

File: src/DynamoCoreWpf/DynamoCoreWpf.csproj [hunk 2]
```diff
@@ -158,12 +158,16 @@
...
+    </Reference>
+    <PackageReference Include="DotNetProjects.Extended.Wpf.Toolkit" Version="5.0.103" Condition=" '$(TargetFramework)' == 'net6.0-windows' "/>
+    <PackageReference Include="Extended.Wpf.Toolkit" version="3.0.0" Condition=" '$(TargetFramework)' == 'net48' ">
```
Comment (mjkkirschner) [inline src/DynamoCoreWpf/DynamoCoreWpf.csproj:165]: is there any reason not to just use the new version?

**Comment by mjkkirschner** [comment]
@sm6srw after this comes in, it would be good to send a PR adding these tests here:
https://git.autodesk.com/Dynamo/DynamoSelfServe/blob/commonTools/scripts/net60_test.ps1

File: src/DynamoCoreWpf/DynamoCoreWpf.csproj (hunk 1 above)
Comment (pinzart90) [reply src/DynamoCoreWpf/DynamoCoreWpf.csproj:162]: @sm6srw could you also move over the net48 specific References into a net48 ItemGroup
Example is not needed for net6 and it causes warnings.
We have very few Assembly References for net6 (as opposed to packagereferences) - only Microsoft.Xaml.Behaviors I think

Comment (sm6srw) [reply src/DynamoCoreWpf/DynamoCoreWpf.csproj:162]: Ok, looks like managed to mess up the merge. This is still WIP. @pinzart Will do.

File: src/Dynamo.All.Net6.sln [hunk 3]
```diff
@@ -156,6 +157,19 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "System", "System", "{F0BFEE
...
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DataBridgeTests", "..\test\Libraries\DataBridgeTests\DataBridgeTests.csproj", "{F0AF3C6E-0E59-4511-A057-79970EA9DC34}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WorkflowTests", "..\test\Libraries\WorkflowTests\WorkflowTests.csproj", "{CEA31792-BA09-431B-8ECE-E2F0354B2150}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CommandLineTests", "..\test\Libraries\CommandLineTests\CommandLineTests.csproj", "{3E84B41E-F229-4BF7-8A0F-3D65BA2D00D1}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestUINodes", "..\test\TestUINodes\TestUINodes.csproj", "{ED1EA294-1BA1-45D6-A669-ECD2E24E21A9}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "UnitsUI", "Libraries\UnitsUI\UnitsUI.csproj", "{C67A7E80-73E6-4BBF-9D3F-DCD86CE306BE}"
```
Comment (aparajit-pratap) [inline src/Dynamo.All.Net6.sln:170]: @mjkkirschner has already migrated this in his [PR](https://github.com/DynamoDS/Dynamo/pull/14137/files#diff-abaedbe9fb07607cdbf540553a7969c6535348335289d81f4518ebf8581d7ff0R140).

File: src/Libraries/GeometryUIWpf/GeometryUIWpf.csproj [hunk 4]
```diff
@@ -89,16 +95,12 @@
     </EmbeddedResource>
   </ItemGroup>
   <Target Name="GenerateFiles" AfterTargets="Build">
```
Comment (aparajit-pratap) [inline src/Libraries/GeometryUIWpf/GeometryUIWpf.csproj:97]: I think this needs to follow the `ResolveSateliteResDeps` target defined in `CS_SDK.props`

File: test/Libraries/CommandLineTests/CommandLineTests.cs [hunk 5]
```diff
@@ -3,13 +3,15 @@
...
 using System.Text.RegularExpressions;
+using System.Threading;
 using System.Xml;
 using Dynamo.Applications;
 using Dynamo.ViewModels;
 using Dynamo.Wpf.ViewModels.Watch3D;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using NUnit.Framework;
+using ProtoCore.AST.ImperativeAST;
```
[the remark this answers is not in the document]
Comment (sm6srw) [reply test/Libraries/CommandLineTests/CommandLineTests.cs:14]: Good catch. Should probably not be there.

File: src/Dynamo.All.Net6.sln (hunk 3 above)
Comment (sm6srw) [reply src/Dynamo.All.Net6.sln:170]: Ok, I missed that. I will wait until that PR merges.

File: src/Libraries/GeometryUIWpf/GeometryUIWpf.csproj (hunk 4 above)
Comment (sm6srw) [reply src/Libraries/GeometryUIWpf/GeometryUIWpf.csproj:97]: Ok, thanks!

File: test/TestUINodes/TestUINodes.csproj [hunk 6]
```diff
@@ -12,18 +15,25 @@
...
+        <DestinationSubDirectory>..\</DestinationSubDirectory>
+    </ReferenceCopyLocalPaths>
   </ItemDefinitionGroup>
   <ItemGroup>
-	  <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="NUnitTestAdapter" Version="2.3.0" ExcludeAssets="all" />
       <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```
Comment (pinzart90) [inline test/TestUINodes/TestUINodes.csproj:26]: why is newtonsoft under net48 ? it should be fine for all net versions

File: test/Libraries/CommandLineTests/CommandLineTests.csproj [hunk 7]
```diff
@@ -10,10 +13,17 @@
     <AssemblyName>CommandLineTests</AssemblyName>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Microsoft.Practices.Prism">
+      <HintPath>..\..\..\extern\prism\Microsoft.Practices.Prism.dll</HintPath>
```
[the remark this answers is not in the document]
Comment (sm6srw) [reply test/Libraries/CommandLineTests/CommandLineTests.csproj:20]: Oops, missed that one. Thanks!

**Review by mjkkirschner — APPROVED** [review]

**Pull request merged**